src/components/Stats/PoolList/style.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core'
+import { colors } from '@static/theme'
+
+const useStyle = makeStyles(() => ({
+  container: {
+    width: 1072,
+    padding: '0 24px',
+    borderRadius: '24px',
+    backgroundColor: `${colors.invariant.component} !important`
+  },
+  pagination: {
+    padding: '20px 0 10px 0'
+  },
+
+  paginationDesign: {
+    '& .MuiPaginationItem-page': {
+      color: `${colors.invariant.light} !important`
+    },
+
+    '& .MuiPaginationItem-page:hover': {
+      color: `${colors.invariant.textGrey} !important`
+    },
+
+    '& .Mui-selected': {
+      color: `${colors.invariant.green} !important`
```

Comment:
This styles are in Pagination component, only you must to do is import component. We try to avoid duplicate code

Reply:
done